CMakeLists.txt

```diff
@@ -1221,6 +1225,17 @@ else()
         set_target_properties(${IMPL_TARGET} PROPERTIES PREFIX "")
     endif()
     target_link_libraries(${IMPL_TARGET} PRIVATE common ${LINKER_FLAGS} ${EXTRA_LIBS} ${MATH_LIB})
+
+    if(APPLE AND ALSOFT_OSX_FRAMEWORK)
+        set_target_properties(${IMPL_TARGET} PROPERTIES
+            FRAMEWORK TRUE
+            FRAMEWORK_VERSION C
+            MACOSX_FRAMEWORK_IDENTIFIER org.opena-soft.openal
+            MACOSX_FRAMEWORK_INFO_PLIST ${OpenAL_SOURCE_DIR}/info.plist
+            PUBLIC_HEADER include/AL/alc.h include/AL/al.h include/AL/alext.h 
```

Comment:
Should this also have `include/AL/efx.h`, since `alext.h` includes it (and some code may try including it directly)?